scripts/build_scripts/build_zips.py

```diff
@@ -103,17 +108,30 @@ def get_build_path(platform, clean_build=False):
   return platform_path
 
 
-def get_cpp_folder_args():
+def get_cpp_folder_args(source_path):
   """Get the cmake args to pass in local Firebase C++ SDK folder.
     If not found, will download from Firebase C++ git repo.
-    
+
+    Args:
+      source_path: root source folder cd back.
+
     Returns:
       cmake args with the folder path of local Firebase C++ SDK. 
       Empty string if not found.
   """
   cpp_folder = os.path.join(os.getcwd(), "..", "firebase-cpp-sdk")
   if os.path.exists(cpp_folder):
-    return "-DFIREBASE_CPP_SDK_DIR=" + os.path.realpath(cpp_folder)
+    cpp_real_path = os.path.realpath(cpp_folder)
+
+    if is_android_build():
```

Comment:
This doesn't seem to be the right place to do this, since this feels it should just be a getter function, but it can randomly trigger an android build. It should probably just be part of the logic that calls cmake and such.

Reply:
Done